dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/UserApi.kt

```diff
@@ -0,0 +1,45 @@
+package org.dataland.datalandbackend.api
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import org.dataland.datalandbackend.entities.DataMetaInformationForMyDatasets
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+
+/**
+ * Defines the restful dataland-backend API regarding user data exchange
+ */
+@RequestMapping("/user")
+@SecurityRequirement(name = "default-bearer-auth")
+@SecurityRequirement(name = "default-oauth")
+interface UserApi {
```

Comment:
Name feels a bit weird.  Seeing this name i would expect to get some interaction with the user data stored on dataland. I would not expect to get company information

Reply:
will change to "UserUploadsApi"

Reply:
ok